axes/models.py

```diff
@@ -53,7 +53,9 @@ class Meta:
 
 class AccessLog(AccessBase):
     logout_time = models.DateTimeField(_("Logout Time"), null=True, blank=True)
-    session_hash = models.CharField(_("Session key hash (sha256)"), default="", blank=True, max_length=64)
+    session_hash = models.CharField(
```

Comment:
This is an unrelated change, please separate reformatting and stylistic things from feature changes into separate commits at the very least.

Reply:
Apologies, I've reverted that now.